config/process_base.config

```
@@ -21,8 +21,8 @@ process{
   withLabel: mem_32 {
     memory = {check_memory(32.GB * task.attempt)}
   }
-  withLabel: mem_128 {
-    memory = {check_memory(128.GB * task.attempt)}
+  withLabel: mem_96 {
+    memory = {check_memory(96.GB * task.attempt)}
```

I kind of feel like we shouldn't just double once we get this large, so here I am suggesting increasing by only 50% at a time. 
```suggestion
    memory = {check_memory(48.GB  + 48.GB * task.attempt)}
```

We should probably update the max_memory along with this... but I think we can do that in `config/profile_ccdl.config`. We can go up to 256GB there. We could also go up to 64 CPUs. (I have a little worry that namespaces might not work quite right with the order things are defined, but I can come back to that.)

---

I went ahead and did that. I looked at the main `nextflow.config` file and we pull in the ccdl profile after we pull in the process config, so maybe it will be okay?